src/h2olog/h2olog.cc

```diff
@@ -272,7 +285,7 @@ int main(int argc, char **argv)
             debug++;
             break;
         case 'l':
-            for (const auto &usdt : available_usdts) {
+            for (const auto &usdt : selected_usdts.empty() ? available_usdts : selected_usdts) {
```

Comment:
I'm not sure if we want to incorporate this change in its current form.

The first problem is that the definition of `-l` changes depending on how `-t` is used. Moreover, the behavior depends on the order those two options are being used. Consider the case where the user specifies `-l -t quicly.*`. Wouldn't the output be surprising?

I'm not sure if we need this, considering the fact that we report an error when the glob does not match anything.

Reply:
That's rather a feature for h2olog developers and I have no objection to drop it. done at 5f57b8a379cbcb38ab99decb74c4f14ecb684551